### ISSUE TITLE
Loading error

### ISSUE THREAD

**Issue by dekrain** [jmoenig/Snap--Build-Your-Own-Blocks#1697]
When I try to load my OS project `New Snap OS!` I get an error:
```
Uncaught Error: missing object
    at SnapSerializer.loadScript
    ...
```

## Version
Latest Snap! Dev [2017.03.22]

## Project
[New Snap! OS](http://snap.berkeley.edu/snapsource/dev/#cloud:Username=DeKrain&ProjectName=New%20Snap!%20OS)

**Comment by brianharvey** [comment]
Did you edit the XML by hand or something?  That's what it sounds like.  Or maybe you somehow deleted a custom block but not some other block that calls it.

**Comment by dekrain** [comment]
No, but I still can open it with stable version (/snapsource/snap.html).
